add-github-organization.sh

```diff
@@ -3,15 +3,9 @@ set -ex
 
 gh repo list $1 --language java --no-archived --source --limit 1000 --json nameWithOwner,defaultBranchRef --template '{{range .}},{{.nameWithOwner}},{{.defaultBranchRef.name}},,,java17,,,,{{"\n"}}{{end}}' | sort > new.csv
 gh repo list $1 --language python --no-archived --source --limit 1000 --json nameWithOwner,defaultBranchRef --template '{{range .}},{{.nameWithOwner}},{{.defaultBranchRef.name}},,,,,,,{{"\n"}}{{end}}' | sort >> new.csv
-# Save the header
-head -n 1 repos.csv > header.csv
-# Copy the data without the header
-tail -n +2 repos.csv > old.csv
-mv header.csv repos.csv
-# Join the old and new data, deduplicate, and sort
-# join merges the data together based upon the columns scmHost,repoName,repoBranch preserving duplicates from the new file
-# awk replaces the empty jdkTool with java17, which is the default, and was stripped by join from the new file
-join -t, -a2 -j1 -o2.2,2.3,2.4,1.5,1.6,1.7,1.8,1.9,1.10,1.11 <(<old.csv awk -F, '{print $1"-"$2"-"$3","$0}' | sort -k1,1 ) <(<new.csv awk -F, '{print $1"-"$2"-"$3","$0}' | sort -k1,1 ) | awk -F ',' -v OFS=',' '$6 == "" { $6 = "java17" }1' | LC_COLLATE=C sort | uniq >> new_deduplicated.csv
-# Join the old and new deduplicate data, and sort
-cat old.csv new_deduplicated.csv | LC_COLLATE=C sort | uniq >> repos.csv
-rm old.csv new.csv new_deduplicated.csv
+gh repo list $1 --language java --archived --source --limit 1000 --json nameWithOwner,defaultBranchRef --template '{{range .}},{{.nameWithOwner}},{{.defaultBranchRef.name}},,,,,,TRUE,archived{{"\n"}}{{end}}' | sort >> new.csv
```

Comment:
why the archived need to be part of this list?

Reply:
My intention with this change is to have an easy way to clear out Jenkins pipelines for projects that will never be able to receive pull requests, because they are archived. With this PR I set the skip column for those to true, such that they will be removed when we next create jobs with the moderne-cli. Seemed like an easy way to achieve a part of https://github.com/moderneinc/moderne-cli/issues/234, and document why we skip repositories while we're at it.

Given that context, is there then still a question of why the archived are part of this list? I might have some difficulty understanding your question.

Reply:
Many of the repositories that are in the `default` repository group are archived. This makes the demo more powerful but those changes will never be merged. I do not think it is convenient to us apply this change now.

Reply:
I understand your concern, but I've not updated any from the default group here (Netflix + Spring projects). I'm also not planning to run this against all organizations or on a schedule (yet), so it should be harmless in that sense right now.

Reply:
Could you elaborate a bit more what are your plans? You have also said "yet" and the issue might occur if we do not know the next steps

Reply:
We can discuss if this is an approach we want to proceed with; seemed low risk to me and convenient in how leverages existing job deletion. Also fits in with the Parser we already have to analyze build tool failures, which again updates the same CSV file.

I've just checked: there are no archived spring-projects repositories, and for the netflix repositories only these four are archived:
```
,Netflix/Fenzo,master,,,,,,TRUE,archived
,Netflix/titus-control-plane,master,,,,,,TRUE,archived
,Netflix/aegisthus,master,,,,,,TRUE,archived
,Netflix/SimianArmy,master,,,,,,TRUE,archived
```
So even if we were to run this for the default set, it would only exclude four repositories, none of which I've ever come across in our demos. Does that remove your concerns with this approach?

As mentioned previously, it would be good to discuss our plans regarding updates on https://github.com/moderneinc/moderne-cli/issues/234#issuecomment-1562951101, so any feedback there is appreciated as well.

Reply:
Checked again. I remembered when I was presenting the workshops having problems because may repositories were archived. At this moment, I do not know if I was in a parallel reality :P

For those 4 it makes sense to me. Thanks for checking.